stories/Input/SelectStory.tsx

```diff
@@ -145,6 +164,45 @@ const SelectStory = () => (
         </Select.Option>
       </Select>
     </div>
+    <div style={{ width: '300px', marginBottom: '1rem' }}>
+      <Select label="Jobs" error={true}>
+        <Select.Option value="accountant">Accountant</Select.Option>
+        <Select.Option value="business development">
+          Business Development
+        </Select.Option>
+        <Select.Option value="software engineer">
+          Software Engineer
+        </Select.Option>
+      </Select>
+    </div>
+    <div style={{ width: '300px' }}>
+      <Select label="Jobs" error="I am an error message 🙀">
+        <Select.Option value="accountant">Accountant</Select.Option>
+        <Select.Option value="business development">
+          Business Development
+        </Select.Option>
+        <Select.Option value="software engineer">
+          Software Engineer
+        </Select.Option>
+      </Select>
+    </div>
+    <div style={{ width: '300px' }}>
+      <Select
+        label="Jobs"
+        error="🎂 I am a custom error message"
+        ErrorComponent={({ children }) => (
```

Comment:
Oh I used `error` instead of `children` because that seems more semantic. No need for the destructuring props here as well because it is just a function returning a component, rather than a component itself.

Reply:
Updated!